add-system.scss

```diff
@@ -0,0 +1,2 @@
+// js generates class '.page__{rootClass}' that wraps the content in your page added to the root tag
+// Found in Routes.js as the rootClass
```

Comment:
does this file need to exist? its all comments

Reply:
I don't think it does right now, but I think it will eventually. I can delete for now though. No need to keep